source/code/plugins/filter_syslog.rb

```diff
@@ -30,6 +30,7 @@ def filter(tag, time, record)
       # Use Time.now, because it is the only way to get subsecond precision in version 0.12.
       # The time may be slightly in the future from the ingestion time.
       record["Timestamp"] = OMS::Common.format_time(Time.now.to_f)
+      record["EventTime"] = OMS::Common.format_time(time)
```

Comment:
After a short investigation it seems that the issue is caused by how Fluentd formats the timestamp from syslog. The Fluentd syslog parser assumes the timestamp is in UTC whereas in reality it might be in a different time zone. 
Time.now.to_f and 'time' are indeed both epoch times but one was taken from the actual local time (and time zone) and the other was parsed (possibly incorrectly) from a string.

I sent you the details of my machine.